### ISSUE TITLE
Wonderful plugin but NO section or anchor links?

### ISSUE THREAD

**Issue by Floffyko** [zioth/dokuwiki-autotooltip#18]
I did not found a way to make a section, an anchor or any relative interlink deep inside a page  functional. It shows only the first section of a page (what I do not find practical for a dictionary or a Bible study). For example the page  `:dw:genesis:cap_1` contains 19 subtexts/verses. How can I make a tooltip separately to each one of the 19 sections/anchors (or whatever needs) to popup separately?

**Comment by zioth** [comment]
Unfortunately, the "automatic" part of autotooltip can't do that. The plugin uses the page's abstract, which is a little bit from the beginning of a page which can be fetched efficiently. If I added the ability to do this with sections, the plugin would get much, much slower.

What you can do is construct the tooltip manually in your wiki:

```
Here is the
<autott>
  <content>first verse</content>
  <tip>In the beginning God created the heavens and the earth. Now the earth was formless and empty, darkness was over the surface of the deep, and the Spirit of God was hovering over the waters.</tip>
  <title>Genesis 1:1</title>
</autott>
of the bible.
```

**Comment by zioth** [comment]
I'll leave this issue open for a while in case I figure out a way to do it efficiently.

**Comment by Floffyko** [comment]
God Idea., thanks. Still, we have to find a solution in a way that the original text to be write only once and then to be easily invoked anywhere it is needed.  Maybe using sqllite?

**Comment by zioth** [comment]
Yes, indexing all pages, and then re-indexing them when they're saved would help solve the problem, but it wouldn't be an easy project. I already have one plugin (fastwiki) which makes significant changes to the way Dokuwiki works. Not sure I have time for another one. :)

For my own reference in case I decide to try something like this, here's the documentation on metadata: https://www.dokuwiki.org/devel:metadata
